client/lcd/test_helpers.go

```diff
@@ -538,11 +538,27 @@ func getKeys(t *testing.T, port string) []keys.KeyOutput {
 	return m
 }
 
+type postKeys struct {
```

Comment:
Aren't these structs _somewhere_ else in our codebase? It seems like we should have structs for calling endpoints defined with those endpoints? Or in a client package with those endpoints?

Reply:
Maybe nothing actionable on this comment

Reply:
I'd suggest to:

1. export the structs - may be useful for testing
2. avoid mentioning HTTP methods such as GET and POST in structure names. Not just a personal preference, also for the sake of consistency with the rest of the codebase.